src/WmPackageServiceProvider.php

```diff
@@ -25,6 +32,15 @@ public function boot()
     {
         parent::boot();
 
+        // Log failed jobs for both queue and horizon
+        Event::listen(QueueJobFailed::class, function (QueueJobFailed $event) {
+            WmLogger::failedJobs()->error($event->exception->getMessage(), ['exception' => $event->exception]);
+        });
+
+        Event::listen(HorizonJobFailed::class, function (HorizonJobFailed $event) {
+            WmLogger::failedJobs()->error($event->exception->getMessage(), ['exception' => $event->exception]);
```

Comment:
valuta (in fase di test) di loggare non solo l'eccezione ma ad esempio gli elementi del costruttore del job. Cioè sarebbe interessante sapere che fallisce il job legato all'EcTrack 453